identity_cache.gemspec

```diff
@@ -31,6 +31,6 @@ Gem::Specification.new do |gem|
     gem.add_development_dependency('cityhash', '0.6.0')
     gem.add_development_dependency('mysql2')
     gem.add_development_dependency('pg')
-    gem.add_development_dependency('stackprof') if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.1.0")
+    gem.add_development_dependency('stackprof') if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new("2.1.0")
```

Comment:
> is frozen in (at least) ruby-1.9.3.p545

Are you still running pre- 2.0. I've been thiking about dropping support for 1.9